### PR TITLE
Add DARS v1 tutorial 

### PR DESCRIPTION
**Pull request by bd-dean**

### CODE REVIEW

**Comment by mvincerx** [comment]
@kpalaniv.  Please add your approval based on our conversation.

File: source/clear-linux/tutorials/dars.rst [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+   scala>
+
+The :command:`--ulimit nofile` parameter is currently required in order to increase the number of open files opened at certain point by the spark engine.
+
+
+Building DARS Images
+********************
+
+If you choose to build your own DARS container images, you can customize them as needed. Use the provided Dockerfile as a baseline. To construct images with Clear Linux, start with a Clear Linux development platform that has the :command:`containers-basic-dev` bundle installed. Learn more about bundles and installing them `swupd guide`_
```
[the remark this answers is not in the document]
Comment (bd-dean) [reply source/clear-linux/tutorials/dars.rst]: UPdated

File: source/clear-linux/tutorials/dars.rst [hunk 2]
```diff
@@ -0,0 +1,91 @@
+.. _dars:
+
+Deep Analytics Reference Stack
+#############################
+
```
[the remark this answers is not in the document]
Comment (bd-dean) [reply source/clear-linux/tutorials/dars.rst]: Always with the "off by one!"  It's updated with latest commit.

**Comment by mvincerx** [comment]
@bd-dean. Please hold off. I'm now overwriting your branch to make corrections. DO NOT PUSH ANYMORE.

**Pull request merged**